test/e2e/policies.test-suite/topology-aware/c4pmem4/test04-dynamic-page-demotion-deprecated-syntax/code.var.sh

```diff
@@ -36,7 +36,7 @@ pages_per_second_per_process="$(awk '
 
 # After how many rounds (seconds) first migrations should be visible.
 first_migrations_visible="$(awk '
-    /PageScanInterval:/{gsub(/[^0-9]/, "", $2); print $2+3}
+    /PageScanInterval:/{gsub(/[^0-9]/, "", $2); print $2+5}
```

Comment:
I suggest to make this 10, in my testing 5 (round 15) was on the limit

```suggestion
    /PageScanInterval:/{gsub(/[^0-9]/, "", $2); print $2+10}
```

Reply:
Or at least `+6` to have some margin for "fluctiations"

Reply:
I agree. Let's increase the margin still.